integration-tests/src/test/java/org/apache/fineract/integrationtests/investor/externalassetowner/InitiateExternalAssetOwnerTransferTest.java

```diff
@@ -154,8 +157,7 @@ private static void setProperFinancialActivity(Account transferAccount) {
     @Test
     public void saleActiveLoanToExternalAssetOwnerWithCancelAndBuybackADayLater() {
         try {
-            GlobalConfigurationHelper.manageConfigurations(REQUEST_SPEC, RESPONSE_SPEC,
-                    GlobalConfigurationHelper.ENABLE_AUTOGENERATED_EXTERNAL_ID, true);
+            globalConfigurationHelper.manageConfigurations(GlobalConfigurationConstants.ENABLE_AUTO_GENERATED_EXTERNAL_ID, true);
```

Comment:
wouldn't a @Before / @After be a better choice for tinkering with the business date in each test separately?

Reply:
Of course there is room for improvement, but rewriting all the tests to do that would have been a much bigger job